xml/ha_storage_protection.xml

```diff
@@ -782,14 +842,49 @@ Timeout (msgwait)  : 10
     node needs to be fenced. </para>
   </sect1>
 
-<!--  <sect1 xml:id="sec.ha.storage.protect.fencing">
-  <title>Storage-based Fencing</title>
-  <para>
-   You can reliably avoid split brain scenarios by using one or more
-   &stonith; Block Devices (SBD), <literal>watchdog</literal> support and
-   the <literal>external/sbd</literal> &stonith; agent.
-  </para>
- </sect1>-->
+  <sect1 xml:id="sec.ha.storage.protect.test">
+   <title>Testing SBD and Fencing</title>
+   <para>To test if the combination of SBD and fencing works, use one or all
+    of the following methods to check them:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>Manually trigger fencing for node <replaceable>NODENAME</replaceable>:</para>
+     <screen>&prompt.root;<command>crm</command> node fence <replaceable>NODENAME</replaceable></screen>
+    </listitem>
+    <listitem>
+     <para>Simulate a SBD failure on the SBD inquisitor and use its process ID:</para>
```

Comment:
Struggling a little bit with the order of this.... How about the following?
Simulate an SBD failure by killing the SBD inquisitor process:
1. Identify the process ID of the SBD inquisitor with the following command: [...]
2. Kill the process: [...]

Also here, would be good to mention the expected result.